test/insured-vesting-v1/insured-vesting-v1.test.ts

```diff
@@ -1057,7 +1063,10 @@ describe("InsuredVestingV1", () => {
 
           const expectedInvalidUsers = [projectWallet, anyUser];
           for (const invalidUser of expectedInvalidUsers) {
-            await expectRevert(async () => insuredVesting.methods.recoverToken(someOtherToken.options.address).send({ from: invalidUser }), OWNER_REVERT_MSG);
+            await expectRevert(
+              async () => insuredVesting.methods.recoverToken(someOtherToken.options.address).send({ from: invalidUser }),
```

Comment:
It doesn't seem to affect the tests, but I'm always surprised we don't need an `await` here too. I guess because `async` functions return a promise regardless is why it works...

Reply:
yes, because the equivalent would be:
```
              async () { return await insuredVesting.methods.recoverToken(someOtherToken.options.address).send({ from: invalidUser 
              })
            }

and usually you do not want `return await`, returning the promise is satisfactory
```